apps/storybook/src/stories/datacom-avatar.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentStoryFn, Meta } from '@storybook/react';
+import { DatacomAvataravatar } from '@datacom/endeavour-react';
+
+export default {
+  title: 'Avatar',
+  component: DatacomAvataravatar,
+  argTypes: {
+    firstName: {
+      name: 'First Name',
+      defaultValue: 'First Name',
+      description: 'Button text',
```

Comment:
If you're adding descriptions, ensure that it's actually describing what this argument controls. For example, this first name argument controls the first name rendered on the component. So I'd write something like 'Controls the first name rendered' (or something similar to that effect.)

Should probably look to fix these up for all the args defined here.